lib/sidekiq/failures/sorted_entry.rb

```diff
@@ -11,7 +11,7 @@ def initialize(parent, score, item)
 
     def retry_failure
       Sidekiq.redis do |conn|
-        results = conn.zrangebyscore(Sidekiq::Failures::LIST_KEY, score, score)
+        results = conn.zrange(Sidekiq::Failures::LIST_KEY, score, score, "BYSCORE")
```

Comment:
The `BYSCORE` argument wasn't added until Redis 6.2, and we support Sidekiq 4+, which allows as far back as Redis 2.8.

I agree that it would be nice to get rid of this warning, I just think we would have to do it in a Redis version aware way.

https://redis.io/commands/zrange/